hw/ip/otbn/data/insns.yml

```diff
@@ -82,6 +82,9 @@ encoding-schemes: enc-schemes.yml
 #  straight-line: A boolean. If true, this instruction has no effect on control
 #                 flow. Optional, default true.
 #
+#  insn-cycles: A positive integer, giving the number of cycles that the
```

Comment:
```suggestion
#  cycles: A positive integer, giving the number of cycles that the
```

Reply:
Drat! Clicked too soon. Will follow up with this.